### PR TITLE
Aggregate and AggregatePart command tests

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
This is WIP. Tests and similar features for testing commands of `Aggregate` are coming.

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
 
-        spineVersion = '0.7.16-SNAPSHOT'
+        spineVersion = '0.7.18-SNAPSHOT'
```
Comment (alexander-yevsyukov) [inline build.gradle]: I needed to advance in two version numbers to solve dependency issues from Projects.

**Comment by alexander-yevsyukov** [comment]
@armiol, PTAL

File: testutil/src/main/java/org/spine3/test/AggregatePartCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    /**
+     * Obtains the aggregate part being tested or {@code Optional#absent()} if
+     * the test object has not been created yet.
+     */
+    protected Optional<P> aggregatePart() {
+        return Optional.fromNullable(aggregatePart);
+    }
+
+    /**
+     * Initialized a test suite with a newly created {@code AggregatePart}.
```
Comment (armiol) [inline testutil/src/main/java/org/spine3/test/AggregatePartCommandTest.java]: I think, it should read `Initializes...`

**Pull request merged**